tests/Agent/IntegrationTests/Shared/WCF/WcfService.cs

```diff
@@ -230,15 +230,16 @@ private async Task<string> DoWork(string value, bool ignoreTransaction, bool thr
             }
 
             ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
-            using (var wc = new WebClient())
+
+            using (var client = new HttpClient())
             {
-                var s = wc.DownloadString(new Uri("https://www.google.com/"));
+                var s = client.GetStringAsync(new Uri("https://www.google.com/")).Result;
```

Comment:
You could probably keep this endpoint as async if you just await this call.

Reply:
Good catch. I'll update that.